go/vt/vttablet/tabletserver/planbuilder/dml.go

```diff
@@ -338,7 +337,7 @@ func analyzeInsertNoType(ins *sqlparser.Insert, plan *Plan, table *schema.Table)
 		for _, col := range ins.Columns {
 			colIndex := table.FindColumn(col)
 			if colIndex == -1 {
-				return nil, fmt.Errorf("column %v not found in table %s", col, table.Name)
+				return nil, vterrors.Errorf(vtrpcpb.Code_NOT_FOUND, "column %v not found in table %s", col, table.Name)
```

Comment:
I think this should still be invalid argument (same comment below). Not Found is more for missing data. I see that it's also misused in other places, which we should eventually fix.

Reply:
Ok I had that thought as well when I've encountered elsewhere. I'll do a quick scan for NOT_FOUNDs and fix